hackathon_site/event/views.py

```diff
@@ -7,16 +7,21 @@
 from django.views.generic.base import TemplateView
 from django.views.generic.edit import FormView
 from django.conf import settings
+from django_filters import rest_framework as filters
+
 
 from hackathon_site.utils import is_registration_open
 from registration.forms import JoinTeamForm
 from registration.models import Team as RegistrationTeam
 
 from rest_framework import generics, mixins
+from rest_framework.filters import SearchFilter, OrderingFilter
```

Comment:
Where do you use OrderingFilter? (I'm only asking because I don't really know how to use it lol)

Reply:
@YuYing-Liang  Ohh it's my mistake. When I copy and paste this line from my previous pull request, I forgot to check and remove this. stupid and awkward mistake again lol

Reply:
Thanks Yuying for pointing that out!